batavia/types/Tuple.js

```diff
@@ -324,6 +324,8 @@ Tuple.prototype.__getitem__ = function(index) {
                 return this[idx]
             }
         }
+    } else if (types.isinstance(index, types.Bool)) {
+        return this[index ? 1 : 0];
```

Comment:
Would it be possible to add a test that fails now, but that this code fixes? This way we don't accidentally break it again. :)

Something as simple as adding a test that checks `[1, 2][True] == 2` would be great.